<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginMenu</class>
 <widget class="QWidget" name="loginMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>381</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Who are you?</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>92</y>
     <width>781</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>781</width>
     <height>321</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>779</width>
      <height>319</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>779</width>
      <height>319</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_user5">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>161</height>
        </size>
       </property>
       <property name="text">
        <string>User 5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_user6">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>161</height>
        </size>
       </property>
       <property name="text">
        <string>User 6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_user4">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>161</height>
        </size>
       </property>
       <property name="text">
        <string>User 4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_user3">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>161</height>
        </size>
       </property>
       <property name="text">
        <string>User 3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_user2">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>161</height>
        </size>
       </property>
       <property name="text">
        <string>User 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_user1">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>161</height>
        </size>
       </property>
       <property name="text">
        <string>User 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_newLogin">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>423</y>
     <width>250</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string>New Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>423</y>
     <width>250</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
